data/client/king-phisher.desktop

```diff
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=King Phisher
 GenericName=King Phisher
-Exec=python /opt/king-phisher/KingPhisher
+Exec=/usr/bin/python3 /opt/king-phisher/KingPhisher
```

Comment:
Is there a reason that we have to have the entire path as opposed to just python3 and letting the PATH environment variable do it's thing?